.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: cpp
+dist: trusty
+matrix:
+  include:
```

Comment:
I suppose it doesn't hurt, but I don't really see need to spend the time running a full matrix here. Seems like picking a single python version for both operating systems would be sufficient. If runtime of the tests ever becomes limiting (as it is with the RDKit, for example), we may want to dial this back and only test a single python version.

Reply:
I'll keep only py3.7 and 3.8. Tests don't take that long for now and we can always slim the matrix if we need to.